src/main/java/org/apache/tools/bzip2/CBZip2InputStream.java

```diff
@@ -29,957 +29,918 @@
-		final boolean[] inUse = dataShadow.inUse;
-		final byte[] pos = dataShadow.recvDecodingTables_pos;
-		final byte[] selector = dataShadow.selector;
-		final byte[] selectorMtf = dataShadow.selectorMtf;
-		
-		int inUse16 = 0;
-		
+
+    private static final int EOF = 0;
+    private static final int START_BLOCK_STATE = 1;
+    private static final int RAND_PART_A_STATE = 2;
+    private static final int RAND_PART_B_STATE = 3;
+    private static final int RAND_PART_C_STATE = 4;
+    private static final int NO_RAND_PART_A_STATE = 5;
+    private static final int NO_RAND_PART_B_STATE = 6;
+    private static final int NO_RAND_PART_C_STATE = 7;
+    private final CRC crc = new CRC();
+    /**
+     * Index of the last char in the block, so the block size == last + 1.
+     */
+    private int last;
+    /**
+     * Index in zptr[] of original string after sorting.
+     */
+    private int origPtr;
+    /**
+     * always: in the range 0 .. 9. The current block size is 100000 * this
+     * number.
+     */
+    private int blockSize100k;
+    private boolean blockRandomised;
+    private int bsBuff;
+    private int bsLive;
+    private int nInUse;
+    private InputStream in;
+    private int currentChar = -1;
+    private int currentState = START_BLOCK_STATE;
+    private int storedBlockCRC, storedCombinedCRC;
+    private int computedBlockCRC, computedCombinedCRC;
+    private int su_count;
+    private int su_ch2;
+
+    // Variables used by setup* methods exclusively
+    private int su_chPrev;
+    private int su_i2;
+    private int su_j2;
+    private int su_rNToGo;
+    private int su_rTPos;
+    private int su_tPos;
+    private char su_z;
+    /**
+     * All memory intensive stuff. This field is initialized by initBlock().
+     */
+    private CBZip2InputStream.Data data;
+    /**
+     * Constructs a new CBZip2InputStream which decompresses bytes read from the
+     * specified stream.
+     * <p>
+     * <p>
+     * Although BZip2 headers are marked with the magic <tt>"Bz"</tt> this
+     * constructor expects the next byte in the stream to be the first one after
+     * the magic. Thus callers have to skip the first two bytes. Otherwise this
+     * constructor will throw an exception.
+     * </p>
+     *
+     * @throws IOException          if the stream content is malformed or an I/O error
+     *                              occurs.
+     * @throws NullPointerException if <tt>in == null</tt>
+     */
+    public CBZip2InputStream(final InputStream in) throws IOException {
+        super();
+
+        this.in = in;
+        init();
+    }
+
+    private static void reportCRCError() throws IOException {
+        // The clean way would be to throw an exception.
+        throw new IOException("crc error");
+
+        // Just print a message, like the previous versions of this class did
+        //System.err.println("BZip2 CRC error");
+    }
+
+    /**
+     * Called by createHuffmanDecodingTables() exclusively.
+     */
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AS1213)